src/useObserver.ts

```diff
@@ -4,15 +4,31 @@ import { printDebugValue } from "./printDebugValue"
 import { isUsingStaticRendering } from "./staticRendering"
 import { useForceUpdate, useUnmount } from "./utils"
 
-export function useObserver<T>(fn: () => T, baseComponentName = "observed"): T {
+export type ForceUpdateHook = () => () => void
+
+export function useObserver<T>(
+    fn: () => T,
+    baseComponentName?: string,
+    customForceUpdateHook?: ForceUpdateHook
```

Comment:
It's probably better to wrap this into a `options` object in case some other stuff comes later so it won't be a breaking change. Keep the `baseComponentName` as is.

Reply:
wrapped